schemas/create-key-pair.yml

```diff
@@ -0,0 +1,13 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: "ssh public key"
+type: object
+properties:
+  value:
```

Comment:
Maybe we call this `keypair` or `publicKey` or something?

Reply:
I was sort of annoyed that I couldn't just get a string as the complete request body, I went vague as a result :)

I'll rename i